core/shared/src/main/scala/scalaz/zio/Exit.scala

```diff
@@ -24,39 +24,39 @@ sealed trait Exit[+E, +A] extends Product with Serializable { self =>
   import Exit._
 
   /**
-   * Retrieves the `A` if succeeded, or else returns the specified default `A`.
+   * Sequentially zips the this result with the specified result or else returns the failed `Cause[E1]`
    */
-  final def getOrElse[A1 >: A](orElse: Cause[E] => A1): A1 = self match {
-    case Success(value) => value
-    case Failure(cause) => orElse(cause)
-  }
+  final def <*>[E1 >: E, B](that: Exit[E1, B]): Exit[E1, (A, B)] = zipWith(that)((_, _), _ ++ _)
 
   /**
-   * Converts the `Exit` to an `Either[Throwable, A]`, by wrapping the
-   * cause in `FiberFailure` (if the result is failed).
+   * Sequentially zips the this result with the specified result discarding the second element of the tuple or else returns the failed `Cause[E1]`
    */
-  final def toEither: Either[Throwable, A] = self match {
-    case Success(value) => Right(value)
-    case Failure(cause) => Left(FiberFailure(cause))
-  }
+  final def <*[E1 >: E, B](that: Exit[E1, B]): Exit[E1, A] = zipWith(that)((_, _), _ ++ _).map(_._1)
```

Comment:
In alphanum names when one method name is a prefix of other methods name, it's placed before.
This is prefix of `<*>`.

Reply:
Thanks, you're right. I've reordered (thanks scaladoc for doing all the work 😅 ).